src/sourmash/sbt.py

```diff
@@ -758,7 +759,9 @@ def load(cls, location, *, leaf_loader=None, storage=None, print_version_warning
             elif storage is None:
                 storage = klass(**jnodes['storage']['args'])
 
-        return loader(jnodes, leaf_loader, dirname, storage, print_version_warning=print_version_warning, cache_size=cache_size)
+        obj = loader(jnodes, leaf_loader, dirname, storage, print_version_warning=print_version_warning, cache_size=cache_size)
+        obj.location = location
```

Comment:
Oops, there were more changes for the `self._location`... It will still work because it is not saved in the SBT and the only place where `self._location` can be set is here, but we ended up with `self.location` being available =]

Reply:
ok, will do test and fix